test/test_loader.js

```diff
@@ -37,7 +37,12 @@ describe('Loader classes', () => {
 
     it('extracts file from ASAR', () => {
       loader.load(test_files.get('asar'));
-      loader.list_files.size.should.equal(60);
+      loader.list_files.size.should.equal(61);
+    });
+
+    it('finds Electron version number in package.json', () => {
```

Comment:
It would be nice to have some other tests checking e.g. if the minimum v0.1.0 fallback is actually used when the version can't be found, but we can add this a later PR too.

Reply:
I definitely agree but I'll have to leave this to someone else.